docs/website/docs/general-usage/data-enrichments/currency_conversion_data_enrichment.md

```diff
@@ -0,0 +1,267 @@
+---
+title:  Currency-conversion data enrichment
+description: Converting the monetary value in one currency to another using the latest market rates.
+keywords: [data enrichment, currency conversion, latest market rates]
+---
+
+# Data enrichment part two: Currency conversion data enrichment
+
+Currency conversion data enrichment means adding additional information to currency-related data.
+Often, you have a data set of monetary value in one currency. It would be best to have these amounts
+in a different currency for different reasons (like reporting, analysis, or operations in a global
+context).
+
+## Setup Guide
```

Comment:
I'm not sure that this section should be called "Setup Guide". It's more Introduction

Reply:
Made it 
Currency conversion process